sdcm/nemesis.py

```diff
@@ -1016,6 +1020,87 @@ def generate_random_parameters_values():
             self.tester.assertLessEqual(len(toppartition_result[sampler]['partitions'].keys()), args['toppartition'],
                                         msg="Wrong number of requested and expected toppartitions for {} sampler".format(sampler))
 
+    def disrupt_network_random_interruptions(self):
+        self._set_current_disruption('NetworkRandomInterruption')
+        if not self.cluster.aws_extra_network_interface:
+            raise ValueError("for this nemesis to work, you need to set `aws_extra_network_interface: True`")
+
+        # get the last 10min max network bandwidth used, and limit  30% to 70% of it
```

Comment:
Why not the average bandwidth? It will really effect the workloads if max is occasional and very higher than the average bandwidth.

Reply:
you've got a point there, I've updated it to use avarage